From c49b22614e4a6f843f8c2395d22bedf335f79fde Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd@luon.net>
Date: Sun, 13 Jul 2014 16:56:16 +0200
Subject: [PATCH 19/37] man: sysusers.d correct default user shell

For the non-root user sysusers uses nologin as the default shell, not
login. Correct the documentation to match the code.
---
 man/sysusers.d.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/sysusers.d.xml b/man/sysusers.d.xml
index 40f8715..b8e5af5 100644
--- a/man/sysusers.d.xml
+++ b/man/sysusers.d.xml
@@ -106,7 +106,7 @@ m authd input</programlisting>
                                         group will be set to the group
                                         bearing the same name. The
                                         user's shell will be set to
-                                        <filename>/sbin/login</filename>,
+                                        <filename>/sbin/nologin</filename>,
                                         the home directory to
                                         <filename>/</filename>. The
                                         account will be created
-- 
1.8.5.5

